### PR TITLE
Update echarts.volkovlabs.io to use visual editor and data sources

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: [#243 ](https://github.com/VolkovLabs/volkovlabs-echarts-panel/issues/243)

### CODE REVIEW

File: echarts.volkovlabs.io/bar.json [hunk 1]
```diff
@@ -62,18 +62,72 @@
...
+            "fields": [
+              {
+                "name": "Category",
+                "type": "string",
+                "config": {},
+                "values": ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
+              },
+              {
+                "name": "Value",
+                "type": "string",
```
Comment (asimonok) [inline echarts.volkovlabs.io/bar.json:121]: Please change number for all places where values are numbers

**Review by asimonok — CHANGES_REQUESTED** [review]

File: echarts.volkovlabs.io/bar.json (hunk 1 above)
Comment (vitPinchuk) [reply echarts.volkovlabs.io/bar.json]: updated

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Comment by mikhail-vl** [comment]
@vitPinchuk Good job, please continue:
1. Update Editor Mode to Visual for updated panels.
2. Convert more charts on Home and other dashboards to showcase as many use cases as possible.
3. I updated Static Data Source properties to enable JavaScript Editor, it can be used to return values. Some charts has a lot of values and it's easier to return them using JavaScript.

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**